app/Http/Middleware/TrustProxies.php

```diff
@@ -24,5 +24,6 @@ class TrustProxies extends Middleware
         Request::HEADER_X_FORWARDED_HOST |
         Request::HEADER_X_FORWARDED_PORT |
         Request::HEADER_X_FORWARDED_PROTO |
-        Request::HEADER_X_FORWARDED_AWS_ELB;
+        Request::HEADER_X_FORWARDED_PREFIX |
```

Comment:
If we're changing to allow any proxy I don't think we need to specify the request headers we need to look for. Changing `$proxies = '*'` should be enough.

Reply:
Laravel docs for reference https://laravel.com/docs/9.x/requests#trusting-all-proxies

Reply:
> If we're changing to allow any proxy I don't think we need to specify the request headers we need to look for. Changing `$proxies = '*'` should be enough.

Yes I know that, but from a security standpoint, it's not the best solution. Ideally, it would be better to have a user configurable option.

Let's go for trust all for now, then you can introduce the env variable later. Ok?

Reply:
> If we're changing to allow any proxy I don't think we need to specify the request headers we need to look for. Changing `$proxies = '*'` should be enough.

From what I understood, the headers are needed so that Laravel recognizes it's behind a reverse proxy, once it knows that, the authorization comes into play. They're not mutually exclusive.